stacks/lemp/service-config/mime.types

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
Yeah.  The downside of vendoring in all of your configs is that you have to vendor in **all** of your configs, which may include a decent amount of stuff you don't really care about maintaining.

Here, I think the overhead is manageable, and it's also pretty easy to update the file from upstream.